Content.Server.Database/Model.cs

```diff
@@ -1119,4 +1123,16 @@ public class RoleWhitelist
         [Required]
         public string RoleId { get; set; } = default!;
     }
+
+    [Table("text_automod")]
+    public sealed class TextAutomodFilter
+    {
+        [Column("filter_id")] public int Id { get; set; }
```

Comment:
Don't prefix this with `filter_`.

Reply:
Fixed in [0947e77](https://github.com/space-wizards/space-station-14/pull/28570/commits/0947e7753c99626fcc74b1155206ea157679284d)